requirements/docs.txt

```diff
@@ -1,4 +1,4 @@
-sphinx>=3.0, <4.0  # fails with sphinx_paramlinks with 4.0.0
+sphinx>=4.0
```

Comment:
lets also update PL :]

Reply:
more over how about this https://github.com/PyTorchLightning/lightning-flash/blob/2113c111cc0dcc7893f9d918f21778497285b7c6/docs/source/conf.py#L83

Reply:
Thanks for the catch! Working on it.